### PR TITLE
Feature/jkw pwa initial

### LINKED ISSUE #42: Create & register ServiceWorker
### Feature Categories:
- React

**Describe the feature/Bug your gonna work on**
- `create-react-app` comes with a built in `ServiceWorker` file, though Robbie recommended we de-activate this.
- [Potential Guide](https://github.com/facebook/create-react-app/issues/2715)

### LINKED ISSUE #67: Edit Manifest file for PWA and metadata
### Feature Categories:

- React 

**Describe the feature/Bug your gonna work on**
- Edit the existing manifest.json file with additional properties/different properties 
- Edit title tag/meta description/any other meta data

### PR DESCRIPTION
**Pull request by jkwest-93**
# Som Timer Pull Request Form
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- After doing our spikes on PWA, we needed to add custom service worker functionality. This PR follows [this articles](https://medium.com/@chinmaya.cp/custom-service-worker-in-cra-create-react-app-3b401d24b875) steps for adding this functionality!
- Additionally, we updated our `manifest.json` file to customize some our app's build. This includes:
     - Adding @Abdeboskey 's custom icons
     - Updating the splash screen color
     - Updating app name & description
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
- All current tests are still passing, via `npm test` in the terminal
- We manually checked for our service worker functionality by running our build & serving the build. To run this build, run `npm run build` in the terminal (from the root directory). This will give instructions for how to `serve` the build- on my machine, I run `serve -s build` (and the terminal will offer some guidance for how to actually serve the build depending on your environment).
- In the dev tools, you can confirm that the service worker is registered (see screenshot below)
- Additionally, you can run a PWA audit with Lighthouse to validate aspects of the PWA (see screenshot below)
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- Closes #42 
- Closes #67 
Screen Shot 2020-10-30 at 6 44 53 PM
Screen Shot 2020-10-30 at 6 46 07 PM

### CODE REVIEW

**Review by Chulstro — APPROVED** [review]
Looks great all! You are rockstars

File: public/index.html [hunk 1]
```diff
@@ -7,9 +7,9 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Som Timer is a timer that cares. This application serves curated wellness content on break intervals. Build in rest so you can focus best!"
```
Comment (sienna-kopf) [inline public/index.html:10]: "a timer that cares" ❤️

File: public/manifest.json [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+  "description": "A timer that cares",
   "icons": [
     {
       "src": "favicon.ico",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
     },
     {
-      "src": "logo192.png",
+      "src": "timerNavIcon.png",
```
Comment (sienna-kopf) [inline public/manifest.json:12]: Yay!! Love the icons you made @Abdeboskey

**Review by sienna-kopf — APPROVED** [review]

**Review by Abdeboskey — APPROVED** [review]
This is one of the most thorough, and professional PRs I've seen. Excellent work, and incredible documentation 👏🏼👏🏼👏🏼

File: public/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkwest-93) [reply public/index.html:10]: I think that's a great addition!

File: src/sw-build.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/* Here, we'll provide instructions to generate the service worker */
```
Comment (rwilliams659) [inline src/sw-build.js:1]: So glad that medium article provided steps that actually work! Stoked we have the service worker set up now

File: public/manifest.json [hunk 4]
```diff
@@ -1,25 +1,27 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Som Timer",
```
Comment (rwilliams659) [inline public/manifest.json:2]: Great job on the manifest, team. This already looks great on mobile and will look even better once this manifest is deployed!

**Review by rwilliams659 — APPROVED** [review]
Great progress on the PWA front! Psyched to see what our next steps are

**Pull request merged**